multinode-demo/gce_multinode.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+command=$1
```

Comment:
I think you'll want a `shift` after `command=$1` so that the `getopts` below won't see `$1` (or is that what `OPTIND=$OPTIND+1` is doing?)

Reply:
Yes, OPTIND+1 is doing the same. If shift is the better form, I can use that.

Reply:
I prefer `shift`